<launch>
  <arg name="echo_mode" default="0"/>  
  <node pkg="udp_receiver" name="udp_receiver_node" type="udp_receiver_node" output="screen">
    <param name="echo_mode" value="$(arg echo_mode)"/>
  </node>
  <!-- rviz -->
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(find udp_receiver)/rviz/udp_receiver.rviz" />
</launch>
